test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -162,7 +163,7 @@ public void EarlyStoppingTest()
         /// <summary>
         /// Multiclass Logistic Regression test.
         /// </summary>
-        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCore))] // netcore3.0 output differs from Baseline
+        [LessThanNetCore30OrNotNetCoreFact]
```

Comment:
We should do a similar thing as you are doing with X64Fact (providing a reason for skip), Because this is an issue that we want to correct in the future and not a limitation of the library (eg onnx runtime is just for 64 bit).